### PR TITLE
pack cargo registry and set env

### PR DESCRIPTION
**Pull request by ukint-vs**
Resolves # .

-
-
-

@reviewer-or-team

### CODE REVIEW

File: .github/workflows/CI.yaml [hunk 1]
```diff
@@ -94,6 +97,7 @@ jobs:
         continue-on-error: true
         run: |
           tar -xf /root/cache/check_cache_${{ github.base_ref }}.tar -C /
+          tar -xf /root/cache/check_cargo_registry_${{ github.base_ref }}.tar -C /
```
Comment (breathx) [inline .github/workflows/CI.yaml:100]:
```suggestion
          tar -xf /root/cache/check_cargo_registry_${{ github.base_ref }}.tar -C
```

File: utils/node-loader/src/batch_pool/api/mod.rs [hunk 2]
```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::redundant_async_block)]
```
Comment (breathx) [inline utils/node-loader/src/batch_pool/api/mod.rs:1]: Is it solvable issue?

**Review by breathx — APPROVED** [review]

File: utils/node-loader/src/batch_pool/api/mod.rs (hunk 2 above)
Comment (ukint-vs) [reply utils/node-loader/src/batch_pool/api/mod.rs:1]: Clippy suggestion is not working.

File: .github/workflows/CI.yaml (hunk 1 above)
Comment (ukint-vs) [reply .github/workflows/CI.yaml:100]: Why

**Pull request merged**